### ISSUE TITLE
`blockComments` rule will comment out valid method line if document comment have blank line.

### ISSUE THREAD

**Issue by r-plus** [nicklockwood/SwiftFormat#1114]
When document comment have blank line, `blockComments` rule will comment out method defined line.

before format
```
/**
  foo

  bar
*/
func foo() {}
```

after format 
```
/// foo
///
/// bar
///func foo() {}
```

SwiftFormat version: 0.49.1
macOS 11.6

**Comment by meherkasam** [comment]
I am also able to reproduce this issue for a function call.

Sample code:

```
/**
 Some comment.

 Some explanation.
 */
SomeFunc(
    SomeParam
)
```

Command run:
```
swiftformat test.swift --rules blockComments
```

Expected output:

```
/// Some comment.
///
/// Some explanation.
SomeFunc(
    SomeParam
)
```

Actual output:

```
/// Some comment.
///
/// Some explanation.
///SomeFunc(
    SomeParam
)
```

**Comment by nicklockwood** [comment]
Thanks for reporting - I'll get it fixed asap.

**Comment by nicklockwood** [comment]
@r-plus @meherkasam fixed in 0.49.2

**Comment by meherkasam** [comment]
Thanks for fixing this, @nicklockwood. Unfortunately, I'm still able to reproduce this issue for a comment in a slightly different format. Here's a simplified example:

```
/**
 Line 1.

 Line 2.

 Line 3.
 */
SomeFunc(someParam)
```

Command to run:
```
swiftformat test.swift --rules blockComments
```

Expected:
```
/// Line 1.
///
/// Line 2.
///
/// Line 3.
SomeFunc(someParam)
```

Actual:
```
/// Line 1.
///
/// Line 2.
///
/// Line 3.
///SomeFunc(someParam)
```

**Comment by nicklockwood** [comment]
@meherkasam d'oh! OK, I'll try again.

**Comment by nicklockwood** [comment]
@meherkasam fixed in 0.49.3
